cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -423,10 +427,7 @@ private void createNamespaces() {
       Throwables.propagate(e);
     }
     LOG.info("Creating and registering {} namespace", Constants.DEFAULT_NAMESPACE);
-    getStore().createNamespace(new NamespaceMeta.Builder()
-                                 .setName(Constants.DEFAULT_NAMESPACE)
-                                 .setDescription(Constants.DEFAULT_NAMESPACE)
-                                 .build());
+    getStore().createNamespace(Constants.DEFAULT_NAMESPACE_META);
```

Comment:
Didn't investigate further after we discussed. @rohitsinha54 do you know why this is needed?